src/ontology/mondo-edit.obo

```diff
@@ -552130,6 +552152,44 @@ xref: UMLS:C0005944 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source=
 is_a: MONDO:0005381 {source="NCIT:C97045"} ! bone disorder
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7006" xsd:anyURI
 
+[Term]
+id: MONDO:0800496
+name: childhood-onset self-limited focal epilepsy syndrome
+def: "A group of conditions characterized by age-dependent occurrence in otherwise normal children. Cognition and neurological evaluation are typically normal. Remission occurs in almost all patients by puberty. Presumed genetic factors have an important role. Seizure semiology and electroencephalographic (EEG) features are specific for each of the syndromes included in this group." [PMID:35503717]
+synonym: "childhood-onset SeLFE" EXACT [https://www.epilepsydiagnosis.org/syndrome/epilepsy-syndrome-groupoverview.html]
+is_a: MONDO:0020072 {source="https://www.epilepsydiagnosis.org/syndrome/epilepsy-syndrome-groupoverview.html"} ! childhood-onset epilepsy syndrome
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8490" xsd:anyURI
+
+[Term]
+id: MONDO:0800498
+name: childhood-onset genetic generalized epilepsy syndrome
+def: "A genetic generalized epilepsy that has an onset during childhood." [https://orcid.org/0000-0001-5208-3432]
```

Comment:
Based on this definition, the parent term should be "genetic generalized epilepsy ".

Reply:
added this